addons/frontend/src/typst-doc.ts

```diff
@@ -28,6 +28,14 @@ export enum PreviewMode {
 
 // let rnd = 0;
 
+function isEmptyPage(elem: Element): boolean {
+  return elem.getAttribute(TypstPatchAttrs.Tid) == 'pqKorb/btZUQlH2NwQkOiBc'
+}
+
+function isReusingEmptyPage(elem: Element): boolean {
+  return elem.getAttribute(TypstPatchAttrs.ReuseFrom) == 'pqKorb/btZUQlH2NwQkOiBc'
+}
+
```

Comment:
can you explain what is an empty(or dummy, per pr description) page?

Reply:
Dummy pages are placeholders without real content. When typst-preview calls `renderSvg` with a window argument, the pages outside of window become dummy. Dummy pages is not necessarily empty, so in semantic we say it dummy, and we check it by whether it is empty.